examples/keyvalue/server/observer.go

```diff
@@ -32,7 +32,7 @@ type Observer struct {
 
 // OnInit is called during service init process. Returning an error halts the init?
 func (o *Observer) OnInit(svc service.Host) error {
-	svc.Logger().Info(
+	svc.Logger().Sugar().Infow(
```

Comment:
Let's defer pending the interface discussion. If we end up not putting a facade in front of logging, we can do away with this API altogether - it's just a roundabout way of giving people access to `zap.L` and `zap.S`, which FX startup configures.